doc/extension_store/README.md

```diff
@@ -0,0 +1,76 @@
+# Proposal for Extension Store Manifest
+
+Example for manifest: [manifest.jsonc](./manifest.jsonc)
+
+## Variable Replacements
+
+* Variables placeholders use the `"My variable 'variableName' has value '${variableName}'"` syntax
+* Naming conventions:
+  * User defined variables are in `lowerCamelCase`
+  * Global constants defined by the system admin:
+    * `BFS_SERVICE`
+    * `BUCKET`
+* User defined variables are defined in the `parameters` section. The Extension Store asks the user to input values for all parameters.
+* Variable placeholders in strings (e.g. BucketFS paths or SQL scripts) are expanded before using them.
+
+## Parameters to Store
+
+The Extension Store must store the following information for an installed VS:
+
+* For each adapter:
+    * ID and version
+        * Required to detect if an older version is already installed
+        * Required to detect if the user tries to install another VS using the same adapter
+    * Paths to installed JARs on BucketFS
+        * File names might change in newer versions.
+        * Needed to delete files from BucketFS during update/delete
+    * Parameters with `scope=adapter`
+        * Required when creating a new VS using the same adapter.
+        * Cannot be changed during update. Maybe hide them?
+* For each Connection / Virtual Schema:
+    * ID and version of the adapter
+        * Required to check if another VS uses the same adapter
+    * Parameters with `scope=connection` that the user entered during the installation process, required for updating the values.
+    * **Don't store** values for fields with `type=password`.
+        * User has to enter a new password when updating.
+
+## Installation Process
+
+1. If no other VS uses this adapter / adapter was not installed before:
+    1. Ask user to enter values for all parameters with `scope=adapter`
```

Comment:
```suggestion
    1. Ask user to enter values for all parameters with `scope=configuration`
```
Remeber: VS are not the only extensions. We already have importers / exporters and DS / ML extensions.